aws-s3/scripts-ccao-data-warehouse-us-east-1/spatial-school.R

```diff
@@ -253,10 +264,16 @@ process_cps_file <- function(s3_bucket_uri, file_year, uri, dist_type) {
         c("schoolname", "school_nm", "short_name", "school_nam")
       )
     ) %>%
-    # GAGE/BOGAN/PHILLIPS/CHICAGO VOCATIONAL and ENGLEWOOD secondary attendance boundaries overlapped in 2020/2021
-    # As GAGE/BOGAN/PHILLIPS/CHICAGO VOCATIONAL were reduced in size due to school closures and a new STEM school in ENGLEWOOD
-    # The new district boundaries are what we're interested in
-    filter(!(str_detect(school_nm, "GAGE|BOGAN|PHILLIPS|CHICAGO VOCATIONAL") & boundarygr != '9, 10, 11, 12')) %>%
+    # GAGE/BOGAN/PHILLIPS/CHICAGO VOCATIONAL and ENGLEWOOD secondary attendance
+    # boundaries overlapped in 2020/2021. As
+    # GAGE/BOGAN/PHILLIPS/CHICAGO VOCATIONAL were reduced in size due to school
+    # closures and a new STEM school in ENGLEWOOD the new district boundaries
+    # are what we're interested in
+    filter(
+      !(str_detect(school_nm, "GAGE|BOGAN|PHILLIPS|CHICAGO VOCATIONAL") &
+        boundarygr != "9, 10, 11, 12" &
+        !(file_year %in% c("2017", "2018")))
```

Comment:
This is the only actual code change.

Reply:
Can you add some context about why the change/logic is needed here?